static/js/components/plot/PhotometryPlot.jsx

```diff
@@ -391,9 +394,19 @@ const PhotometryPlot = ({
       return newPoint;
     });
 
+    // Set the range of the plot to be 2% larger than values or if t0 is set, start the range from t0
```

Comment:
the 2% is for the mag, doesn't seem related to the time axis. This comment is a little bit confusing